conan.cmake

```diff
@@ -83,6 +84,22 @@ function(conan_cmake_settings result)
 endfunction()
 
 
+function(conan_cmake_detect_gnu_libcxx result)
+    get_directory_property(defines DIRECTORY ${CMAKE_SOURCE_DIR} COMPILE_DEFINITIONS)
+    foreach(define ${defines})
+        if(define STREQUAL "_GLIBCXX_USE_CXX11_ABI=0")
+            set(${result} libstdc++ PARENT_SCOPE)
+            return()
+        endif()
+    endforeach()
+    if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "5.1")
```

Comment:
Hm, that's unfortunate that gcc 5.1+ might not have libstdc++11 available even though it's supposed to...

The usage of `get_directory_property` checks for definitions (added via `add_definitions`) to see if `_GLIBCXX_USE_CXX11_ABI` has been defined to 0 (which is the way to tell GCC to use libstdc++ instead of libstdc++11).